app/serializers/remote_search_form_serializer.rb

```diff
@@ -0,0 +1,31 @@
+class RemoteSearchFormSerializer < ActiveModel::Serializer
+  self.root = false
+
+  attributes :sort, :query
+
+  def sort
+    'registered_name'
+  end
+
+  def query
+    {
+      'filtered' => {
+        'filter' => {
+          'in' => {
+            'other_advice_methods' => remote_advice_method_ids
+          }
+        }
+      }
+    }
+  end
+
+  private
+
+  def remote_advice_method_ids
```

Comment:
Yep. Although it feels slightly dirty in its approach - we know that the only consistent attribute across environments would be the `order` attribute. I say we key off of that with predicates comparing the `order` attribute in the reference data model instance to a constant defined within the scope of the same reference data class -- so at least this knowledge lives in the right place.

We'll need to add `order` sequences to the requisite factories too. Just leaving that in here for completion's sake :smile: